sources/common/toddl.go

```diff
@@ -52,58 +52,63 @@ type ToDdl interface {
 // the Spanner schema to conv.SpSchema.
 func SchemaToSpannerDDL(conv *internal.Conv, toddl ToDdl) error {
 	for _, srcTable := range conv.SrcSchema {
-		spTableName, err := internal.GetSpannerTable(conv, srcTable.Name)
+		SchemaToSpannerDDLHelper(conv, toddl, srcTable)
+	}
+	internal.ResolveRefs(conv)
+	return nil
+}
+
+func SchemaToSpannerDDLHelper(conv *internal.Conv, toddl ToDdl, srcTable schema.Table) error {
```

Comment:
After dropping the tables synthetic primary keys , ToSoure, ToSpanner and ToSourceFkIndex and ToSpannerFkIndex still appear in the session file

Reply:
These deleted fields get restore becuase in **getSummary** API, we loop over source-table. This issue is fixed by looping over spanner-table.